cpp/oneapi/dal/backend/primitives/objective_function/test/fixture.hpp

```diff
@@ -339,15 +401,16 @@ class logloss_test : public te::float_algo_fixture<Param> {
         return logloss;
     }
 
-    double naive_logloss(const ndview<float_t, 2>& data_host,
-                         const ndview<float_t, 1>& params_host,
-                         const ndview<std::int32_t, 1>& labels_host,
-                         const float_t L1,
-                         const float_t L2,
-                         bool fit_intercept) {
+    float_t naive_logloss(const ndview<float_t, 2>& data_host,
+                          const ndview<float_t, 1>& params_host,
+                          const ndview<std::int32_t, 1>& labels_host,
+                          const float_t L1,
+                          const float_t L2,
```

Comment:
We have an agreement to pass by value without specifying it as `const` in function arguments.